se/Mortgage/v0/schema/MortgageOffering.yaml

```diff
@@ -0,0 +1,111 @@
+"$id": "https://open-broker.org/se/Mortgage/v0/schema/MortgageOffering"
+"$schema": "http://json-schema.org/draft-07/schema#"
+
+title: MortgageOffering
+description: An offer for a mortgage
+type: object
+additionalProperties: false
+required:
+  - brokerReference
+  - offer
+
+properties:
+  brokerReference:
+    title: "The reference used by the broker for the application resulting in this offer"
+    "$ref": "definitions/Reference#/reference"
+  offer:
+    "$ref": "#/definitions/Offer"
+
+definitions:
+  Offer:
+    type: object
+    additionalProperties: false
+    required:
+      - effectiveInterestRate
+      - nominalInterestRate
+      - arrangementFee
+      - termFee
+      - invoiceFee
+
+    properties:
+      effectiveInterestRate:
+        type: string
+        pattern: "^[0-9]+(\\.[0-9]+)?$"
+        title: Effective interest rate (APR)
+        description: |
```

Comment:
It might also be nice-to-have a short description explaining what effective and nominal interest rate actually means. Not necessary though.

Reply:
Yes, it would be nice :thinking: If I can describe it well enough without being misleading in any way I might consider to add it in the future. I believe (most of) the users of this specification will be familiar enough with the terms.